### PR TITLE
Encoder viewing programs minor fix

### PR DESCRIPTION
**Pull request by ctrl-z-9000-times**
From https://github.com/htm-community/nupic.cpp/pull/448#discussion_r281889515

Add comments explaining about oversampling the inputs.
Use more samples than needed to avoid aliasing & artifacts.

Reduced the oversample factor from 5 to 2.  Before with 5 all of the
diagrams were stretched out along the X axis, excessively so.  Now they
are more compact, without losing any fidelity.

### CODE REVIEW

File: py/src/nupic/encoders/rdse.py [hunk 1]
```diff
@@ -93,7 +93,8 @@
         n_samples = int(args.maximum - args.minimum + 1)
     else:
         n_samples = (args.maximum - args.minimum) / enc.parameters.resolution
-        n_samples = int(round( 5 * n_samples ))
+        oversample = 2 # Use more samples than needed to avoid aliasing & artifacts.
+        n_samples  = int(round( oversample * n_samples ))
```
[the remark this answers is not in the document]
Comment (ctrl-z-9000-times) [reply py/src/nupic/encoders/rdse.py]: The encoder does not  encode a range of numbers.  It encodes a single number at a time. If the users wants to encode a range of numbers then they need a `for` loop to call the encoder with each input sample.  Choosing good numbers to encode is the users problem.

[the remark this answers is not in the document]
Comment (breznak) [reply py/src/nupic/encoders/rdse.py]: >  breznak 6 hours from now 

woot! I'm commenting from the future! :man_astronaut: 

> Choosing good numbers to encode is the users problem.
 
yes, one at a time. Encoding well any given number is RDSEs problem. So this 'oversample' thing is a hack just for the visualisations to look well, not affecting any ability to encode any numbers?

[the remark this answers is not in the document]
Comment (ctrl-z-9000-times) [reply py/src/nupic/encoders/rdse.py]: > So this 'oversample' thing is a hack just for the visualisations to look well, not affecting any ability to encode any numbers?

Yes

**Comment by breznak** [comment]
@dkeeney looks like we're again running 2 instances of Windows CI, for /pr & /branch. You've fixed it once in the config, do you recall what, why it occurs again?

**Pull request merged**